e2e/test_plugin_installations.py

```diff
@@ -19,38 +19,39 @@ def test_sample_plugins_installed(self):
             "windows-slaves": "1.0",
             "ldap": "1.11",
             "ant": "1.2",
-            "matrix-auth": "1.1",
-            "translation": "1.10",
-            "mailer": "1.11",
-            "pam-auth": "1.1",
-            "antisamy-markup-formatter": "1.1",
-            "maven-plugin": "2.7.1",
+            "matrix-auth": "1.2",
+            "translation": "1.12",
+            "mailer": "1.16",
+            "pam-auth": "1.2",
+            "antisamy-markup-formatter": "1.3",
+            "maven-plugin": "2.8",
             "ssh-slaves": "1.9",
-            "javadoc": "1.1",
-            "subversion": "1.54",
+            "javadoc": "1.3",
+            "subversion": "2.4.5",
             "external-monitor-job": "1.4",
-            "cvs": "2.11",
+            "cvs": "2.12",
             # The following plugins installed via test_data/plugins.yml
-            "ec2": "1.29",
-            "ghprb": "1.34.0",
+            "ec2": "1.28",
+            "ghprb": "1.22.4",
             "job-dsl": "1.45",
             # The following plugins should be installed as dependencies for
             # the plugins installed via test_data/plugins.yml
             "node-iterator-api": "1.1",
             "github": "1.9.1",
             "plain-credentials": "1.1",
-            "credentials": "1.21",
-            "matrix-project": "1.6",
+            "credentials": "1.24",
+            "matrix-project": "1.4.1",
             "ssh-agent": "1.3",
-            "github-api": "1.82",
```

Comment:
I'm curious about why some of these plugins are the version they are. Almost all of them match build Jenkins (including GHPRB). But I'm seeing the following differences on Build Jenkins:
```
github-api 1.69
git 2.4.0
git-client 1.19.0
mapdb-api 1.0.6.0
```